README.md

```diff
@@ -202,6 +202,10 @@ color_thresholds:
   - value: 4
     color: "#0000ff"
 ```
+i.e. if value is between "0" (including this value) & "1.33333" - color is "#ff0000",
```

Comment:
This looks quite "like a block", when rendered. Could you maybe make it a list?  I think, that would be more readable.

Reply:
Could you give an example?